specification/logic/resource-manager/Microsoft.Logic/stable/2019-05-01/logic.json

```diff
@@ -9210,6 +9339,9 @@
         "sku": {
           "$ref": "#/definitions/IntegrationAccountSku",
           "description": "The sku."
+        },
+        "identity": {
```

Comment:
I just want to double confirm. Does it mean the identity also needs to be set while creating integration account?

Reply:
This is actually not supported. Let me remove this. thanks for catching.